database/cassandra/README.md

```diff
@@ -25,7 +27,7 @@ system_schema table which comes with 3.X
 | `sslcert` | | Cert file location. The file must contain PEM encoded data. |
 | `sslkey` | | Key file location. The file must contain PEM encoded data. |
 | `sslrootcert` | | The location of the root certificate file. The file must contain PEM encoded data. |
-| `sslmode` | | Whether or not to use SSL (disable\|require\|verify-ca\|verify-full) |
```

Comment:
I believe these options were copied from [postgres](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNECT-SSLMODE).
It's probably worth documenting that here (probably just link to the postgres docs) since the meaning for each `sslmode` is not obvious.

Reply:
Do you just want the docs updated with a comment inline here? I was thinking about going ahead and knocking that out so this can PR can be merged in.

Reply:
Adding a link to the postgres docs is sufficient